recipe/meta.yaml

```diff
@@ -20,6 +20,7 @@ requirements:
     - python
     - numpy
     - six
+    - enum34
```

Comment:
I think this may only be needed for py27

Reply:
Thanks! Comment added.